apps/greencheck/tests/test_azure_cloud_importer.py

```diff
@@ -0,0 +1,114 @@
+import pytest
+import pathlib
+import ipaddress
+import json
+from io import StringIO
+
+from django.core.management import call_command
+from apps.greencheck.management.commands import update_azure_ip_ranges
+from apps.accounts.models import Hostingprovider
+from apps.greencheck.models import GreencheckIp
+from apps.greencheck.management.commands.update_azure_ip_ranges import GREEN_REGIONS
+
+from django.conf import settings
+
+
+@pytest.fixture
+def hosting_provider():
+
+    # oregon, *rest = [region for region in GREEN_REGIONS if region[1] == "us-west-2"]
```

Comment:
Would you please remove this part? It's for the other provider

Reply:
It has been removed.

As there is only one region, I've also left that part out. 
Do you think we should name this service "Azure" or "Microsoft", or something else?

Reply:
Also, for now I have used 123 as hosting id. Can this be any arbitrary number?